delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/DeltaDirectives.scala

```diff
@@ -102,13 +133,16 @@ trait DeltaDirectives extends RdfMarshalling {
   def requestMediaType: Directive1[MediaType] =
     extractRequest.flatMap { req =>
       val ct = new MediaTypeNegotiator(req.headers)
```

Comment:
It is simpler to keep the logic and just add the */* mediaRange to the acceptedMediaRanges to the list, as follow:

```scala
(ct.acceptedMediaRanges :+ `*/*`).foldLeft[Option[MediaType]](None) { ... }
```

Since the RFC2616 says
> If no Accept header field is present, then it is assumed that the client accepts all media types

Reply:
Won't it break the compatibility for clients ?

Reply:
Well, it is what the RFC states, so I assume it is the right thing to do.

Reply:
I think it's fine to follow the RFC... I can't see how this would break the compatibility.